resources/shellHelperFunctions.sh

```diff
@@ -201,5 +201,9 @@ verify_npm_setup() {
 
   pushd "$KEYMAN_ROOT" > /dev/null
   npm ci
+
+  # Configure Web browser-engine testing environments.  As is, this should only
+  # make changes when we update the dependency, even on our CI build agents.
+  playwright install
```

Comment:
Use of the new setup uses [playwright](https://github.com/microsoft/playwright), which operates by using browser engines directly.  It works by downloading specific versions of the relevant browser engines... and this command performs that download.

Note that it essentially performs a global install, outside of the repository space.  So, once it's fired successfully once, the engines are essentially cached until we change `playwright` version.  `playwright` supposedly also detects when previously-downloaded engines are no longer referenced, so it _should_ remove old versions after an upgrade.  (I am a _touch_ worried about it "thrashing" them when an upgrade is pending review and/or branch-hopping, though.)

Reply:
This should not be done in the build script.

Reply:
After reviewing what `playwright install` does, I am more comfortable with leaving this line in. I do think it belongs outside `verify_npm_setup` because it's only for web-based, not node-based projects, so a dev on Keyman Developer won't want it, for example.